2018/python/test_file_generator.py

```diff
@@ -2866,11 +2874,16 @@ def q604(self):
 		path = self.quality_path
 		ts = self.ts_df.copy()
 		lar = self.lar_df.copy()
-		lar.state = lar.state.map(lambda x: random.choice(list(self.crosswalk_data.stateCode)))
+		lar.state = lar.state.map(lambda x: self.state_abbrev[random.choice(list(self.crosswalk_data.stateCode))])
```

Comment:
Please use underscores rather than camel case to match the design pattern present.

Reply:
Sure - would this mean changing `crosswalk_data` to `crossWalkData`?

Reply:
No, it would be crosswalk_data.stateCode to crosswalk_data.state_code.

Reply:
Sure, the geographic flat file headers have not been changed in the refactor, and I think that changing those names could be a separate issue with a separate pull request.

Reply:
Can you point me to the geo file and the headers? I see several files in the dependency folder, but don't see a header file or anything with camel case.

Reply:
Is [this](https://github.com/cfpb/hmda-test-files/blob/master/2019/python/configurations/geographic_data.yaml) the header you're referencing?

Since you're already making changes to the state codes portion, I'd prefer to close out the camel case issue in this PR

Reply:
Sounds good - the changes have been made in the most recent commits.